Firmware/MotorControl/main.cpp

```diff
@@ -21,6 +21,11 @@ SystemStats_t system_stats_ = { 0 };
 
 Axis *axes[AXIS_COUNT];
 
+// Added for environments where new is not defined
```

Comment:
Bad idea on embedded. Why have you added this? I see nowhere that you're using it, nor should you use it given the extremely tight memory conditions of embedded systems.

Reply:
It would not compile for me without explicitly defining "new"

Reply:
You have issues with your builder/compiler then, what OS/tup version/arm gcc version are you using?

Reply:
Ubuntu 18, standard arm-none-eabi, using make, not tup.

Reply:
Yeah you have something wrong... malloc/new work fine.  What version of arm-none-eabi-gcc?  8.x is not supported.

Reply:
7.3.0